spec/README.md

```diff
@@ -160,19 +161,27 @@ Type: `String`
 
 The associated language written in English.
 
+###### metadata.languageNative
+
+Required  
+Type: `String`
+
+The associated language written in the native language.
+
+
 ###### metadata.variant
 
 Optional  
 Type: `String`
 
 The associated language variant written in English, if applicable. For example, if the [IETF language tag](https://www.w3.org/International/articles/language-tags/) was `de_AT`, this entry would include the full name of the variant country, `Austria` for the `AT` variant.
 
-###### metadata.native
+###### metadata.variantNative
 
 Required  
```

Comment:
Shouldn't this be "Optional" since the `metadata.variant` is optional?

Reply:
My fault, fixed.